brands/amenity/pharmacy.json

```diff
@@ -710,6 +710,18 @@
       "name": "London Drugs"
     }
   },
+  "amenity/pharmacy|Longs Drugs~(Hawaii)": {
```

Comment:
Is the disambiguating suffix needed? Any Longs Drugs location outside of Hawaii would’ve changed its name by now, so there’s only one “Longs Drugs” chain to include in the index. Then again, 2fc39b458d5ec02f6f91443f892ee2909feee402 added a disambiguator to Noah’s Bagels even though there’s only one Noah’s Bagels chain, so I’m no longer certain about the criteria for disambiguators.

Reply:
It’s probably overkill and I have no opposition to removing it.  I was being extra cautious because while the company was acquired by CVS in 2008, I don’t know when the rebranding was completed.  I was worried about the confusion around potential old POIs which have long since rebranded but were not updated in OSM (I haven’t checked if any actually exist).  My use of the disambiguator was meant as an extra precaution because we don’t have a US “stateCode” as we do “countryCode”.

Reply:
I actually hand cleaned up all the ones left on the west coast a while back. They should all be CVS at this point.

Reply:
That works for me.  I agree with this reasoning and reverted the disambiguation.

Reply:
Per the discovery of a "Longs Drugs" with ~20 locations in the Southeast US, I re-added the "~Hawaii" (comment linked below).  Even if there is currently only one Longs Drugs in the NSI, this will help to prevent confusion for mappers in SC, NC, and GA.

https://github.com/osmlab/name-suggestion-index/pull/2795#issuecomment-502680399